### PR TITLE
[caffe2ImporterTest] Add eq unittest caffe2ImporterTest

### PR DESCRIPTION
**Pull request by gcatron**
*Description*: Added EQ node test to caffe2ImporterTest. This checks that the node is imported and the graph is the expect shape.
*Testing*: Run caffe2ImporterTest caffe2.EQ1D passes.
#985

### CODE REVIEW

**Comment by rdzhabarov** [comment]
Do not worry about ci/circleci for now, the work on it is in progress (expect failure).
Also, make sure to squash commits into a single one, for example, by doing ```git rebase -i``` and then ```git push -f origin HEAD`

File: tests/unittests/caffe2ImporterTest.cpp [hunk 1]
```diff
@@ -1013,3 +1013,60 @@ TEST(caffe2, batchBoxCox) {
...
+  // will not depend on anyting from the loader.
+  {
+    Caffe2ModelLoader caffe2LD(NetDescFilename, NetWeightFilename, {"X", "Y"},
+                               {&X.getType(), &Y.getType()}, *F);
+    output = caffe2LD.getSingleOutput();
+    ctx.allocate(mod.getPlaceholders());
+    updateInputsByName(ctx, &mod, {"X", "Y"}, {&X, &Y});
+  }
+
+  // Check that the shape of the output matches that of the expected output.
```
Comment (rdzhabarov) [inline tests/unittests/caffe2ImporterTest.cpp]: seems like a stale comment

File: tests/unittests/caffe2ImporterTest.cpp [hunk 2]
```diff
@@ -1013,3 +1013,60 @@ TEST(caffe2, batchBoxCox) {
...
+  // We have 1 EQ and 1 Output.
+  EXPECT_EQ(F->getNodes().size(), 2);
+
+  // Check that the graph has the expected shape (EQ -> Save),
+  // starting from the output.
+  auto *saveNode = getSaveNodeFromDest(output);
+  auto *EQN = llvm::dyn_cast<CmpEQNode>(saveNode->getInput());
+  ASSERT_TRUE(EQN);
+
+  // With have two inputs and one output.
```
Comment (rdzhabarov) [inline tests/unittests/caffe2ImporterTest.cpp]: grammar

**Comment by gcatron** [comment]
Accidentally pulled in an unrelated commit, creating a clean PR and closing this one. 
Comments were addressed and fixed in new PR.

**Pull request closed without merging**